code/game/machinery/vending.dm

```diff
@@ -1589,5 +1601,241 @@
+					/obj/item/clothing/suit/armor/bulletproof/serbian/tan = 1800,
+					/obj/item/clothing/suit/armor/bulletproof/serbian/full = 1900,
+					/obj/item/clothing/suit/armor/bulletproof/serbian/full/green = 1900,
+					/obj/item/clothing/suit/armor/bulletproof/serbian/full/tan = 1900,
+					/obj/item/clothing/head/armor/helmet = 1000, 
+					/obj/item/clothing/suit/storage/vest/merc = 2300,
+					/obj/item/clothing/suit/storage/vest/merc/full = 2500
+				)
+/obj/machinery/vending/clothing_vendor
+	name = "Forever 2421"
+	desc = "All your clothes shopping needs, and you dont even need to go to the mall!"
+	product_slogans = "That is SOOO Your look~"
+	product_ads = "New Clothes, New you!; That is totally your look.; Did you try the ripped jeans look?"
+	icon_state = "generic" // Placeholder till I can make a new one -R
+	no_criminals = FALSE
+	products = list( 
+					/obj/item/clothing/under/genericr = 5,
+					/obj/item/clothing/under/genericw = 5,
+					/obj/item/clothing/under/genericb = 5,
+					/obj/item/clothing/under/neon/red = 5,
+					/obj/item/clothing/under/neon/blue = 5,
+					/obj/item/clothing/under/neon/yellow = 5,
+					/obj/item/clothing/under/neon = 5,
+					/obj/item/clothing/under/leisure/pullover = 5,
+					/obj/item/clothing/under/leisure/white = 5,
+					/obj/item/clothing/under/leisure = 5,
+					/obj/item/clothing/under/dress = 5,
+					/obj/item/clothing/under/dress/blue = 5,
+					/obj/item/clothing/under/dress/red = 5,
+					/obj/item/clothing/under/suit_jacket = 5,
+					/obj/item/clothing/under/suit_jacket/red = 5,
+					/obj/item/clothing/under/tuxedo = 5,
+					/obj/item/clothing/under/shorts = 5,
+					/obj/item/clothing/under/shorts/red = 5,
+					/obj/item/clothing/under/shorts/green = 5,
+					/obj/item/clothing/under/shorts/blue = 5,
+					/obj/item/clothing/under/shorts/black = 5,
+					/obj/item/clothing/under/shorts/grey = 5
+					)
+	
+	prices = list(	
+					/obj/item/clothing/under/genericr = 200,
+					/obj/item/clothing/under/genericw = 200,
+					/obj/item/clothing/under/genericb = 200,
+					/obj/item/clothing/under/neon/red = 200,
+					/obj/item/clothing/under/neon/blue = 200,
+					/obj/item/clothing/under/neon/yellow = 200,
+					/obj/item/clothing/under/neon = 200,
+					/obj/item/clothing/under/leisure/pullover = 200,
+					/obj/item/clothing/under/leisure/white = 200,
+					/obj/item/clothing/under/leisure = 200,
+					/obj/item/clothing/under/dress = 200,
+					/obj/item/clothing/under/dress/blue = 200,
+					/obj/item/clothing/under/dress/red = 200,
+					/obj/item/clothing/under/suit_jacket = 200,
+					/obj/item/clothing/under/suit_jacket/red = 200,
+					/obj/item/clothing/under/tuxedo = 200,
+					/obj/item/clothing/under/shorts = 200,
+					/obj/item/clothing/under/shorts/red = 200,
+					/obj/item/clothing/under/shorts/green = 200,
+					/obj/item/clothing/under/shorts/blue = 200,
+					/obj/item/clothing/under/shorts/black = 200,
+					/obj/item/clothing/under/shorts/grey = 200
```

Comment:
Teeny-tiny gripe that you don't actually need to address, but I feel like variation in prices is nice, even if these are all mechanically just cosmetic. After all, who's selling a tux for the same cost as a hoodie?

Reply:
Its intended to be cheap, to encourage people using the vendors.

Reply:
that being said, I will tweak